app/screens/thread/index.js

```diff
@@ -7,17 +7,22 @@ import {connect} from 'react-redux';
 import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
 
 import {selectPost} from 'mattermost-redux/actions/posts';
+import {getMyCurrentChannelMembership, makeGetChannel} from 'mattermost-redux/selectors/entities/channels';
 import {makeGetPostIdsForThread} from 'mattermost-redux/selectors/entities/posts';
-import {getMyCurrentChannelMembership} from 'mattermost-redux/selectors/entities/channels';
 
 import Thread from './thread';
 
 function makeMapStateToProps() {
     const getPostIdsForThread = makeGetPostIdsForThread();
+    const getChannel = makeGetChannel();
 
     return function mapStateToProps(state, ownProps) {
+        const channel = getChannel(state, {id: ownProps.channelId});
```

Comment:
Not something to change now, but we could add a simple `getChannel` function that isn't a selector since memoization on this is pretty pointless since this is equivalent to `state.entities.channels.channels[id]`

Reply:
we need the freaking display_name here, that's why I'm using this one